_dev/js/back/src/components/banner/paypal-status.vue

```diff
@@ -0,0 +1,159 @@
+<!--**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *-->
+<template>
+  <div class="card" v-if="this.displayLiveStep">
+    <div class="card-body d-flex">
+      <div class="d-flex">
+        <img src="@/assets/images/cb.png" alt="cb" />
+      </div>
+      <div class="flex-grow-1">
+        <div
+          v-if="
+            cardPaymentIsActive === 'IN_REVIEW' ||
+              cardPaymentIsActive === 'LIMITED' ||
+              cardPaymentIsActive === 'NEED_MORE_DATA'
+          "
+        >
+          <p class="mb-0">
+            {{ $t('banner.paypalStatus.congrats') }}
+          </p>
+          <h3 class="mb-4">
+            {{ $t('banner.paypalStatus.oneMoreThing') }}
+          </h3>
+        </div>
+        <div v-else>
+          <h3 class="mb-0">
+            {{ $t('banner.paypalStatus.allSet') }}
+          </h3>
+          <p class="mb-4">
+            {{ $t('banner.paypalStatus.confirmation') }}
+          </p>
+        </div>
+
+        <p class="mb-3 text-muted">
+          <img
+            src="@/assets/images/baseline-check_circle.png"
+            style="width:25px;"
```

Comment:
I guess this image is not needed as we already have material icons in the project in which we can find similar icon in svg. We can keep a high quality and lightweight icon.

See https://material.io/resources/icons/?icon=check_circle&style=baseline

Reply:
On another note, please avoid the style inline in your code for better maintainability.

Declare your style in the `<style>` block like in https://github.com/PrestaShopCorp/ps_checkout/blob/0f951120d134ca6f6f1d1328f287e946d6a7e10e/_dev/src/components/panel/express-checkout.vue#L182-L191

Reply:
done on next commit